.changeset/rude-beers-promise.md

```diff
@@ -0,0 +1,7 @@
+---
+"remix": patch
+"@remix-run/serve": patch
```

Comment:
```suggestion
```

Reply:
remix and remix-serve depend on server-runtime iirc, so should be in here imo. on the other hand all packages are fixed on each other for releasing, so it's fine having just one 🤷